docs/en/advanced/Scalability-Tuning.md

```diff
@@ -17,86 +17,97 @@ aims to enumerate a list of such parameters which should be tuned for the scale
 
 ### Heap Size
 
-The Alluxio master heap size directly controls the total number of files that can fit into the
-master memory. Control the heap size by setting JVM options for the Alluxio master and secondary
-master processes. For example, to set the heap size limit to `256GB`, modify `alluxio-env.sh` as
-follows.
+The Alluxio master heap size controls the total number of files that can fit into the master memory.
+The following JVM options, set in `alluxio-env.sh`, determine the respective maximum heap sizes for the
+Alluxio master and standby master processes to `256 GB`:
+
 ```properties
 ALLUXIO_MASTER_JAVA_OPTS+=" -Xms128g -Xmx256g "
 ALLUXIO_SECONDARY_MASTER_JAVA_OPTS+=" -Xms128g -Xmx256g "
 ```
 
-Note:
-
 * As a rule of thumb set the min heap size to half the max heap size.
-* Each thread spawned by the master JVM requires off heap space determined by the thread stack
-size. When setting the heap size, ensure that you have enough memory allocated for off heap storage.
-For example, to spawn `50000` threads with a default thread stack size of `1MB` ensure you
-have at least `50GB` of off-heap space available.
+* Each thread spawned by the master JVM requires off heap space determined by the thread stack size.
+When setting the heap size, ensure that there is enough memory allocated for off heap storage.
+For example, spawning `50000` threads with a default thread stack size of `1MB` requires at least
+`50 GB` of off-heap space available.
 
 ### Thread Pool Size
+
 An executor pool is used on the master to handle concurrent client requests. If you expect a large
 number of concurrent clients communicating with the master, tune the thread pool size by modifying
 the following properties. The actual thread count depends on the maximum concurrency expected on the
 cluster. For example, if spawning `10000` concurrent tasks (clients) each with a client thread
 pool size of `4`, set the master thread pool max to greater than `40000`.
+
+TODO: wait aren't these properties for this master <-> worker communication, not master <-> client communication?
```

Comment:
halp

Reply:
the naming of properties is bad. it is actually for master<->client communitcation